rpcwebsocket.go

```diff
@@ -681,22 +684,41 @@ func (m *wsNotificationManager) subscribedClients(tx *dcrutil.Tx, clients map[ch
 		}
 
 		for i, output := range msgTx.TxOut {
-			_, addrs, _, err := txscript.ExtractPkScriptAddrs(output.Version,
+			watchOutput := true
+			sc, addrs, _, err := txscript.ExtractPkScriptAddrs(output.Version,
 				output.PkScript, m.server.server.chainParams)
 			if err != nil {
 				// Clients are not able to subscribe to
 				// nonstandard or non-address outputs.
 				continue
 			}
+			if stake.DetermineTxType(msgTx) == stake.TxTypeSStx && i&1 == 1 {
```

Comment:
Not a big deal, but this is a fairly expensive call and the tx isn't changing.  I think it could go up on line 671.

```Go
isTicket := stake.DetermineTxType(msgTx) == stake.TxTypeSStx
for q ,c := range clients { 
....
if isTicket && i&1 == 1 {
```